README.md

```diff
@@ -32,15 +32,15 @@ Demonstrate your understanding of this week's concepts by answering the followin
 
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your Team Lead
 
-1. If you were to describe semantic HTML to the next cohort of students, what would you say?
+1. If you were to describe semantic HTML to the next cohort of students, what would you say? HTML is a way to layout graphically text or content in a webpage.
 
```

Comment:
That is correct, another thing to consider is accessibility  -> https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML